### ISSUE TITLE
All Vivint devices became unavailable after HA update

### ISSUE THREAD

**Issue by hydroculator** [natekspencer/hacs-vivint#118]
### The problem

Just updated to HA 2023.9.2 and all Vivint devices are unavailable. They all say "This entity is no longer being provided by the vivint integration. If the entity is no longer in use, delete it in settings."    

### What version of Home Assistant Core are you running?

2023.9.2

### What was the last working version of Home Assistant Core?

2023.9.1

### Anything in the logs that might be useful?

```txt
Not adding entity with invalid device info: Invalid device info {'default_manufacturer': 'Vivint', 'identifiers': {('vivint', '#)}, 'manufacturer': None, 'model': 'FIREFIGHTER_AUDIO_DETECTOR', 'name': 'Fire Detector', 'sw_version': 0, 'via_device': ('vivint', '#)} for 'vivint' config entry: device info needs to either describe a device, link to existing device or provide extra information.
```

**Comment by natekspencer** [comment]
This was resolved in [2023.8.0](https://github.com/natekspencer/hacs-vivint/releases/tag/2023.8.0), please make sure you have upgraded to the latest release

**Comment by natekspencer** [comment]
> > Just updated to HA 2023.9.2...

You have to update to the latest Vivint integration release as well. HA core does not update hacs-vivint.
